next.config.js

```diff
@@ -43,6 +43,10 @@ const moduleExports = {
     firebaseAppId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
     firebaseMeasurementId: process.env.NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID,
     facebookAppId: process.env.NEXT_PUBLIC_FACEBOOK_APP_ID,
+    nearNetworkId: process.env.NEAR_NETWORK_ID ?? 'testnet',
```

Comment:
is this used @ical10 ? look likes `NEAR_NETWORK_ID` deleted from .env-template

Reply:
readded these fields back into `.env-template`, yes these are necessary env vars

Reply:
got it!